torchbenchmark/models/hf_Whisper/__init__.py

```diff
@@ -0,0 +1,20 @@
+from torchbenchmark.util.framework.huggingface.model_factory import HuggingFaceModel
+from torchbenchmark.tasks import SPEECH
+import torch
+
+class Model(HuggingFaceModel):
+    task = SPEECH.RECOGNITION
+    # https://cdn.openai.com/papers/whisper.pdf Says for large-v2 they trained on 1024 batch sizes, with 16 GPUs
+    DEFAULT_EVAL_BSIZE = 64
+    
+    def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
+        super().__init__(name="hf_Whisper", test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+        self.feature_size = 80
+        self.sequence_length = 3000
+        input_features = torch.randn(size=(self.batch_size, self.feature_size, self.sequence_length),device=self.device)
+        self.example_inputs = {"input_features": input_features.to(self.device)}
```

Comment:
Since we are wrapping the model in a different way, we need to implement customized `get_module()` here, similar to the upstream code: https://github.com/MaanavD/benchmark/blob/116df9cb937b6921d16eba34fc504776bb40a6ee/torchbenchmark/util/framework/huggingface/model_factory.py#L110

The reason we need `get_module()` is because this API is being used by our downstream benchmarking script: https://github.com/pytorch/pytorch/blob/main/benchmarks/dynamo/torchbench.py#L358
and it requires `model(*example_input)` runs successfully.